makefile

```diff
@@ -0,0 +1,28 @@
+.SILENT:
+.DEFAULT:
+
+MAKEFILES_DIR = makefiles
+
+help:
+	if [ $(firstword $(MAKECMDGOALS)) == $@ ]; then \
+		echo 'Run `make <command> help` for more specific help' | fold -s; \
+		echo ''; \
+		echo 'commands:'; \
+		for file in $(wildcard $(MAKEFILES_DIR)/*.makefile); do \
```

Comment:
Hmm, I don't have a `wildcard` on debian, and aptitude isn't showing any results.

If I understand intent, wouldn't this work with a simple `for file in "${MAKEFILES_DIR}/*.makefile"; do`